src/citrine/resources/branch.py

```diff
@@ -1,18 +1,25 @@
 import functools
+import warnings
 from typing import Iterator, Optional, Union
 from uuid import UUID
 
+from deprecation import deprecated
+
 from citrine._rest.collection import Collection
 from citrine._rest.resource import Resource
 from citrine._serialization import properties
 from citrine._session import Session
+from citrine._utils.functions import migrate_deprecated_argument
 from citrine.exceptions import NotFound
 from citrine.resources.data_version_update import BranchDataUpdate, NextBranchVersionRequest
 from citrine.resources.design_workflow import DesignWorkflowCollection
 from citrine.resources.experiment_datasource import (ExperimentDataSourceCollection,
                                                      ExperimentDataSource)
 
 
+LATEST_VER = "latest"  # Refers to the most recently created branch version.
```

Comment:
I'm presuming this is intended to be importable as a keyword.  If this is wrong, we should add a leading underscore.

There's a similar constant defined for "latest" in predictor.py.  Are these programmatically tied on the back end?

Reply:
Yes, it is intended to be public, like the similar one in predictors. But no, they're not tied together on the backend.